### ISSUE TITLE
Date parsing not consistent.

### ISSUE THREAD

**Issue by RussellMcOrmond** [crkn-rcdr/Access-Platform#318]
I note from looking at https://access.canadiana.ca/object/69429/m0fj29883036

"Status: published on 31/12/1969, 19:33:40"

Dates can be in two formats, and this parsing should probably happen within Lapin to make consistent.

Related: I'm regularly needing to convert the seconds-since to a string when using Fauxton for checking the state of data.  Having the human readable version of the date be what is in the database would be ideal, even if the numeric version is used in all the applications.

**Comment by SaschaAdler** [comment]
I agree. I propose we use the ISO 8601 date specification. All of it, for maximal interoperability and minimal "having to think about it". If anything has a hard time with fractional seconds (which, btw, Solr doesn't, at least as of the link you shared) or non-UTC time zones, we take care of it then.

If we restrict ourselves unnecessarily, we then have to focus on ES, CouchDB, Perl, Solr, MySQL, Blazegraph, Fortran, or whatever.
